src/perf-goggles.js

```diff
@@ -191,13 +244,24 @@ export const queryPerformanceData = async (
       },
     };
   } else {
-    const subtestsMeta = await querySubtests(seriesConfig, parentInfo.parentSignatureHash);
+    const subtestsMeta = await querySubtests(
+      seriesConfig,
+      parentInfo.parentSignatureHash
+    );
+
     subtestsMeta[parentInfo.parentSignatureHash] = parentInfo;
-    const subtestsData = await fetchSubtestsData(seriesConfig, subtestsMeta, timeRange);
-    Object.keys(subtestsData).forEach((hash) => { perfData[hash] = subtestsData[hash]; });
+    const subtestsData = await fetchSubtestsData(
+      seriesConfig,
+      subtestsMeta,
+      timeRange
+    );
+
+    Object.keys(subtestsData).forEach(hash => {
+      perfData[hash] = subtestsData[hash];
+    });
   }
 
   return perfData;
 };
 
-export default queryPerformanceData;
+export { queryPerformanceData };
```

Comment:
Please leave with `default`.
The only reason there's any other exports is because of testing purposes.
For all intents and purposes external consumers should only be using the default exported function.